TAO/tao/tao.mpc

```diff
@@ -60,6 +60,12 @@ project(TAO_Core_idl) : tao_versioning_idl_defaults, gen_ostream, install, pidl_
     WStringSeq.pidl >> AnyTypeCode/WStringSeqA.h AnyTypeCode/WStringSeqA.cpp
   }
 
+  IDL_Files {
+    idlflags += -Sci -Gse -Gata --idl-version 4
+    Int8Seq.pidl >> AnyTypeCode/Int8SeqA.h AnyTypeCode/Int8SeqA.cpp
```

Comment:
New Int8SeqA/UInt8SeqA needs to be added to AnyTypeCode

Reply:
That's not in scope. Can I just remove those parts of the line?

Reply:
Are the files generates but empty? Then just add them, we shouldn’t have generated files which we don’t compile